control/bgp.yaml

```diff
@@ -116,4 +121,103 @@ components:
           format: uint32
           maximum: 3600
           default: 30
-          x-field-uid: 2
\ No newline at end of file
+          x-field-uid: 2
+
+    Action.Protocol.Bgp.LoadRoutes:
+      description: >-
+        Loads routes to be advertised for a given peer in a given format.
+      type: object
+      properties:
+        configs:
+          type: array
+          items:
+            $ref: '#/components/schemas/Action.Protocol.Bgp.LoadRoutes.Config'
+          x-field-uid: 1
+
+    Action.Protocol.Bgp.LoadRoutes.Config:
+      description: >-
+        Configuration for loading routes to be advertised for a given peer in a given format.
+      type: object
+      properties:
+        routes:
+          $ref: '#/components/schemas/Action.Protocol.Bgp.Routes'
+          x-field-uid: 1
+        peer_names:
+          description: |-
+            The names of configured BGP peers from which the routes shall be advertised.
+          type: array
+          items:
+            type: string
+            minItems: 1
+          x-constraint:
+            - '/components/schemas/Bgp.V4Peer/properties/name'
+            - '/components/schemas/Bgp.V6Peer/properties/name'
+          x-field-uid: 2
+        use_best_routes:
+          description: |-
+            When set to false, all routes available shall be advertised.
+          type: boolean
+          default: true
+          x-field-uid: 3
+        retain_next_hop:
+          description: |-
+            When set to false, the next hop address present against each route to be advertised
+            is replaced with next hop address of the corresponding test port.
+          type: boolean
+          default: false
+          x-field-uid: 4
+        
+
+    Action.Protocol.Bgp.Routes:
+      description: >-
+        Routes to be advertised for a given peer in a given format.
+      type: object
+      properties:
+        choice:
+          description: Format of routes to be advertised.
+          type: string
+          x-field-uid: 1
+          x-enum:
+            v4_routes:
```

Comment:
Do we have any use cases for these (v4_routes & v6_routes) options? If not, can it be added later, once required?

Reply:
The use case is driven by development cycle.
- One should be able to load routes in OTG format if implementation of other formats are not available
- Test ports do not have the capability to understand vendor specific formats, only Controller does, hence controller to test port communication would require sending imported routes in OTG format